Website/src/rendering/src/assets/css/components/NewsList.css

```diff
@@ -31,6 +31,10 @@
         }
       }
 
+      .image-container {
+        @apply h-40 overflow-hidden;
```

Comment:
The issue does not seem to be fixed for all screen sizes. See here at 1289px wide:
![image](https://user-images.githubusercontent.com/2000200/145608004-504b04fb-f714-4bf8-a65f-fc2ae27d709d.png)

There is also another issue unrelated to the work item where the images are not as wide as their container on smaller screen sizes. If it is possible to fix it at the same time, it would be great!
![image](https://user-images.githubusercontent.com/2000200/145608353-e9f88e98-b53b-45e5-a28e-5a41434643fc.png)

Reply:
Both issues should be fixed now.